charts/capi-runtime-extensions/templates/cni/calico/manifests/tigera-operator-configmap.yaml

```
@@ -12,4 +12,5 @@ data:
 kind: ConfigMap
 metadata:
   creationTimestamp: null
-  name: '{{ .Values.hooks.CalicoCNI.defaultTigeraOperatorConfigMap.name }}'
+  name: '{{ .Values.hooks.CalicoCNI.crsStrategy.defaultTigeraOperatorConfigMap.name
+    }}'
```

I think changed by accident?
```suggestion
  name: '{{ .Values.hooks.CalicoCNI.crsStrategy.defaultTigeraOperatorConfigMap.name}}'
```

---

This is due to addons sync using kustomize and kustomize wrapping the value as it is now longer. This still works thanks to YAML semantics, even if it doesn't look very nice.

---

Ahh got it 👍